src/lib/default.nix

```diff
@@ -5,7 +5,7 @@ let
 
   # nixpkgs lib
   base = nixpkgs.lib // builtins;
-  pkgs = { inherit styx; inherit (nixpkgs) runCommand writeText; };
+  pkgs = { inherit styx; inherit (nixpkgs) runCommand writeText writeScript bash; };
```

Comment:
See the previous comment on why I need writeScript. bash is for the script shebang